docs/proposals/mvp-dashboard-design.md

```diff
@@ -10,16 +10,37 @@ Last updated: 2017-01-12
 
 ## Abstract
 
-A short summary of the proposal.
+We propose a dashboard/server that allows for the automated retrieval,
+archive, format, and display of performance tests run in the performance lab.
+This tool will allow our team to investigate performance issues and facilitate
+the sharing of lab results with other teams.
 
 ## Background
 
-An introduction of the necessary background and the problem being solved by
-the proposed change.
+Viewing performance test results from the performance lab as it trends across
+time is tedious and repetitive. To view this data, the following actions must
+take place:
+
+1. Run Performance Test
+1. Retrieve report file
+1. Format data so that it can be entered into a database
+1. Insert data into database
+1. Query the database for timerange of tests
```

Comment:
Made changes to better represent the actions taken for reviewing performance test results.